kitchen.yml

```diff
@@ -3,7 +3,7 @@ driver:
   name: vagrant
 
 provisioner:
-  name: chef_zero
+  name: chef_infra
```

Comment:
We can remove this as infra is the default. Though I don't mind being explicit if you really want.

Reply:
AFAIK zero is technically "deprecated" and `chef_infra` is the preferred provisioner to be used.

Reply:
So just remove the provisioner `name` completely here?